src/Illuminate/Database/Eloquent/Factories/Sequence.php

```diff
@@ -37,6 +39,16 @@ public function __construct(...$sequence)
         $this->count = count($sequence);
     }
 
+    /**
+     * Get the current count of the sequence items.
+     *
+     * @return int
+     */
+    public function count()
```

Comment:
@acurrieclark @taylorotwell The `int` return typehint is missing so this will throw a deprecation on PHP 8.1. Since this is completely new code there's no need to add the `ReturnTypeWillChange` attribute when the return type can be added instead.

https://wiki.php.net/rfc/internal_method_return_types

Reply:
Thanks. I fixed that.